src/main/java/org/mule/extensions/jms/internal/connection/provider/activemq/ActiveMQConnectionProvider.java

```diff
@@ -134,6 +187,10 @@ private void executeCleanup(Connection connection) throws Exception {
     }
   }
 
+  private String getAdviceMessage(String library) {
+    return "Validate that the Mule Application has the required library: \"" + library + "\" as a Shared Library";
```

Comment:
in case of missing libs, add the brokerUrl to the error message, thus the user can quickly identify any missing/extra flag